benefit/data/sheets/CaseField/CaseField.json

```diff
@@ -245,6 +245,7 @@
   {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "ID": "adjournCaseAdditionalDirections", "Label": "Additional directions", "FieldType": "Collection", "FieldTypeParameter": "TextArea", "SecurityClassification": "PUBLIC"},
   {"LiveFrom": "20/08/2019", "CaseTypeID": "Benefit", "ID": "adjournCasePreviewDocument", "Label": "Preview Adjournment", "HintText": "All documents must be PDF formatted", "FieldType": "Document", "SecurityClassification": "PUBLIC"},
   {"LiveFrom": "20/08/2019", "CaseTypeID": "Benefit", "ID": "adjournCaseGeneratedDate", "Label": "Adjourn case generated date", "FieldType": "Date", "SecurityClassification": "PUBLIC"},
+  {"LiveFrom": "23/11/2022", "CaseTypeID": "Benefit", "ID": "adjournmentInProgress", "Label": "Adjournment in progress?", "FieldType": "YesOrNo", "SecurityClassification": "PUBLIC"},
```

Comment:
Move this to the snl nonprod sheet please

Reply:
You need to add authorisations (in the snl non-prod sheet) otherwise, this won't work.

Reply:
done